pkg/store/sandbox/metadata.go

```diff
@@ -55,6 +55,8 @@ type Metadata struct {
 	NetNSPath string
 	// IP of Pod if it is attached to non host network
 	IP string
+	// AdditionalIPs of the Pod if it is attached to no host network
```

Comment:
s/no/non/

nit: Maybe put this new field at the bottom?

Reply:
I think `IP` and `AdditionalIPs` should better be together. :)

Reply:
yeah and json marshal is by field name not number.. so it's ok.. just normal convention to add new fields to bottom. Convention based on old tech where serialization was binary though.